File: GrowthWare_VB/GrowthWareFramework/ModelObjects/MGroupRoles.vb
Namespace ModelObjects
	<Serializable(), CLSCompliant(True)> _
	Public Class MGroupRoles
		Private mADD_UP_BY As Integer
		Private mSE_SEQ_ID As Integer = -1
		Private mGROUP_SEQ_ID As Integer = -1
		Private mRoles As String

		Public Property ADD_UP_BY() As Integer
			Get
				Return mADD_UP_BY
			End Get
			Set(ByVal value As Integer)
				mADD_UP_BY = value
			End Set
		End Property

		Public Property SE_SEQ_ID() As Integer
			Get
				Return mSE_SEQ_ID
			End Get
			Set(ByVal value As Integer)
				mSE_SEQ_ID = value
			End Set
		End Property

		Public Property GROUP_SEQ_ID() As Integer
			Get
				Return mGROUP_SEQ_ID
			End Get
			Set(ByVal value As Integer)
				mGROUP_SEQ_ID = value
			End Set
		End Property

		Public Property Roles() As String
			Get
				Return mRoles
			End Get
			Set(ByVal value As String)
				mRoles = value.Trim
			End Set
		End Property
	End Class
End Namespace

File: GrowthWare_CS/GrowthWareFrameworkModel/Profiles/MGroupRoles.cs

using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Web;
namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public class MGroupRoles
	{
		private int m_ADD_UP_BY;
		private int m_SE_SEQ_ID = -1;
		private int m_GROUP_SEQ_ID = -1;

		private string mRoles;
		public int ADD_UP_BY {
			get { return m_ADD_UP_BY; }
			set { m_ADD_UP_BY = value; }
		}

		public int SE_SEQ_ID {
			get { return m_SE_SEQ_ID; }
			set { m_SE_SEQ_ID = value; }
		}

		public int GROUP_SEQ_ID {
			get { return m_GROUP_SEQ_ID; }
			set { m_GROUP_SEQ_ID = value; }
		}

		public string Roles {
			get { return mRoles; }
			set { mRoles = value.Trim(); }
		}
	}
}
